src/provisioner/provisioner.py

```diff
@@ -84,17 +112,17 @@ def config_apply(solution_conf_url: str, cortx_conf_url: str = None, \
                     val = Cipher.encrypt(cipher_key, val)
                     # decoding the byte string in val variable
                     Conf.set(CortxProvisioner._solution_index, key, val.decode('utf-8'))
-            CortxProvisioner.config_apply_cortx(cortx_conf)
+            CortxProvisioner.solution_config_apply(cortx_conf, cortx_release)
 
     @staticmethod
-    def config_apply_cortx(cortx_conf):
+    def solution_config_apply(cortx_conf, cortx_release):
```

Comment:
Can we rename this instead to the actual action i.e. "apply_cortx_config" because we do apply cortx conf.

Reply:
Done.